R/comparison.R

```diff
@@ -0,0 +1,93 @@
+rcmdcheck_comparison <- function(old, new) {
+  stopifnot(is.list(old))
+  stopifnot(inherits(new, "rcmdcheck"))
+
+  # Generate data frame of problems
+  old_df <- do.call(rbind, lapply(old, as.data.frame, which = "old"))
```

Comment:
Do you think we should keep supporting all the comparisons with CRAN? It seems less important now